e2e-python/files/tests/acceptance/great_expectations/test_preparation/pre_requisites.py

```diff
@@ -0,0 +1,185 @@
+import os
+import boto3
+import json
+from pathlib import Path
+
+"""
+
+This is an example of what you could do as a pre-requisite before executing your great expectations tests, in this example we deploy an athena database
+with two tables: person and address.
+
+In the post_requisite.py we will delete this athena database
```

Comment:
This is an example of what you could do as a pre-requisite before executing your great expectations tests. This is intended to prepare your data sets or even trigger your ETL pipelines. 

In this  specific example we deploy a sample athena database
with two tables: person and address that will be used on the Demo test cases.

In the post_requisite.py we will delete this athena database